src/app/sidenav/sidenav.component.scss

```diff
@@ -0,0 +1,118 @@
+//sidenav style 
+.sidenav {
+  background: #f1e8f1;

```

Comment:
Still the same issues I have found in SCSS files

Reply:
Changes done.